draft-ietf-ppm-dap.md

```diff
@@ -561,20 +560,20 @@ Aggregators. This scheme has two important properties:
   measurement from which it was generated.
 
 * Aggregators can compute secret shares of the aggregate result by aggregating
-  their shares locally into "aggregate shares", which may later be combined
-  into the aggregate result.
-
-Note that some VDAFs allow measurements to be aggregated multiple times, each
-time with a different aggregation parameter; however, DAP only allows each
-measurement to be aggregated once. Similarly, some VDAFs produce aggregate
-values which depend on the order in which the measurements are aggregated;
-however, DAP only supports VDAFs whose aggregation results are independent of
-the order in which measurements are aggregated (see {{Section 4.4.1 of !VDAF}}).
+  their shares locally into "aggregate shares", which may later be merged into
+  the aggregate result.
+
+DAP is not compatible with all VDAFs. Some VDAFs produce aggregate
+values which depend on the order in which the measurements are aggregated. DAP
```

Comment:
might: This is situation is very hypothetical
values -> results: consistency with terminology
```suggestion
DAP is not compatible with all VDAFs. Some VDAFs might produce aggregate
results which depend on the order in which the measurements are aggregated. DAP
```